spec/actionsBuild.js

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
```

Comment:
What about this? Did you try the regex based check that we discussed?

Reply:
I'm getting a weird error from while getting the diff.
Here's the error log: https://pastebin.com/yb6iqnUY

The command I'm running is `git diff --no-index dist_file_path new_build_file` from node.
Running this normally gives the expected output

Reply:
Hmm, did you look into what can be causing the error? Can we try adding a catch block to the promise to handle any errors like this? One other way could be to use a custom function which accepts two string args and finds the diff between them.

Reply:
Following discussion via chat, we resolved to stick to just the pre commit hook.